services/web/server/src/simcore_service_webserver/login/decorators.py

```diff
@@ -23,4 +23,6 @@ async def wrapped(*args, **kwargs):
     return wrapped
 
 
-__all__ = ["login_required",]
+__all__ = [
```

Comment:
Since you are refactoring, please use tuples here instead of lists

Reply:
Out of curiosity, is there a reason why?

Reply:
it's faster...